### ISSUE TITLE
[Adaptive Cards - iOS - ColumnSet.SelectAction.json]: Application get closed when user activate 'More Activities' link using double tap on it.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8416]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Elements' control and activate it.
6. Navigate to 'ColumnSet.SelectAction.json' and activate it.
7. Card will appear.
8. Navigate to 'More Activities' link and double tap on it.
9. Observe the issue.

**Actual Result:**
Application gets closed when user activate 'More Activities' link using double tap on it.

**Expected Result:**
Application should not be closed when user activate 'More Activities' link using double tap on it. After pressing link, link should redirect to the expected page.

**Note:**
1. The issue is also repro with 'Resume All' link on same card.
2. This issue also repro's without voice over with finger touch. 

**User Experience:**
User will be impacted, if application gets closed after activating control, it will be time consuming for user to navigate back to their expected place.

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/228123065-b0ece997-f9bf-4d8b-96f6-d8770fedbd61.mov

### Card JSON

```JSON
NA
```

**Comment by Raisul123** [comment]
Adding A11yUsable tag to functional bug for tracking purpose

**Comment by jwoo-msft** [comment]
@Raisul123,
I'm closing this issue because the cards under the Scenario should be tested. I will update the sample app. The reason for this is that those cards are only meant for internal testing, we never intended to use in outside of our testing and meant for functional testing for features.
thank you.

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as per the above comment.
